terraform-digitalocean/main.tf

```diff
@@ -0,0 +1,43 @@
+#Terraform Configuration
+
+variable "DO_API_TOKEN" {
+  description = "DigitalOcean Access Token"
+}
+
+variable "TESTNET_NAME" {
+  description = "Name of the cluster/testnet"
+  default = "tf-testnet1"
+}
+
+variable "ssh_keys" {
+  description = "SSH keys provided in DigitalOcean to be used on the nodes"
+  # curl -X GET -H "Content-Type: application/json" -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" "https://api.digitalocean.com/v2/account/keys"
+  default = ["9495227"]
```

Comment:
DigitalOcean has an identifier for SSH keys added on the website. The identifier is not presented on the website but you can call into the API to get the list. (The comment shows you how.) These SSH keys are used to connect to the created node using SSH.